oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
This likely doesn't need to be a public module. I'd prefer it to be `_pkce`.

In all honesty, this module is absolutely tiny and can probably just be in-lined in the flow class.